packages/shared/src/components/FeedItemComponent.tsx

```diff
@@ -147,6 +147,9 @@ export default function FeedItemComponent({
         : PostTypeToTag[item.post.type] ?? ArticlePostCard;
       return (
         <PostTag
+          enableSourceHeader={
+            feedName !== 'squad' && item.post.source.type === 'squad'
```

Comment:
`source?.` because it might be undefined per TS, not sure why it is not erroring.

Reply:
Looks like we have this disabled https://www.typescriptlang.org/tsconfig#strict not sure why because a lot of checks like `strictNullChecks` (which would show error in this case since `source` is optional) are not working. cc @rebelchris

Reply:
Fixed the above and made note of this for the next web meeting